src/mumble_exe/mumble_exe.cpp

```diff
@@ -170,6 +325,16 @@ int main(int argc, char **argv) {
 #endif  // DEBUG
 
 int APIENTRY wWinMain(HINSTANCE instance, HINSTANCE prevInstance, wchar_t *cmdArg, int cmdShow) {
+	if (!CleanupEnvironmentVariables()) {
```

Comment:
Lots of code duplication with main() by now. Up to ```GetProcAddress``` for the entry point they seem to be identical. Probably time to extract a function.